lib/kafo_parsers/puppet_strings_module_parser.rb

```diff
@@ -130,6 +132,23 @@ def docs
 
     private
 
+    def capture3(command)
+      env_vars = command.include?('/opt/puppetlabs') ? clean_env_vars : ::ENV
+      Open3.capture3(env_vars, *command, :unsetenv_others => true)
```

Comment:
Is it needed to expand the var? I thought capture3 accepted an array.

Reply:
```
	24: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/rake-13.0.0/lib/rake/rake_test_loader.rb:5:in `<main>'
	23: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/rake-13.0.0/lib/rake/rake_test_loader.rb:5:in `select'
	22: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/rake-13.0.0/lib/rake/rake_test_loader.rb:17:in `block in <main>'
	21: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/rake-13.0.0/lib/rake/rake_test_loader.rb:17:in `require'
	20: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:5:in `<top (required)>'
	19: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:6:in `<module:KafoParsers>'
	18: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `describe'
	17: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `class_eval'
	16: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:7:in `block in <module:KafoParsers>'
	15: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `describe'
	14: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `class_eval'
	13: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:15:in `block (2 levels) in <module:KafoParsers>'
	12: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:15:in `each'
	11: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:16:in `block (3 levels) in <module:KafoParsers>'
	10: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `describe'
	 9: from /home/ehelms/workspace/upstream/kafo_parsers/.bundle/ruby/2.5.0/gems/minitest-5.12.2/lib/minitest/spec.rb:87:in `class_eval'
	 8: from /home/ehelms/workspace/upstream/kafo_parsers/test/kafo_parsers/puppet_strings_parser_test.rb:17:in `block (4 levels) in <module:KafoParsers>'
	 7: from /home/ehelms/workspace/upstream/kafo_parsers/lib/kafo_parsers/puppet_strings_module_parser.rb:15:in `parse'
	 6: from /home/ehelms/workspace/upstream/kafo_parsers/lib/kafo_parsers/puppet_strings_module_parser.rb:15:in `new'
	 5: from /home/ehelms/workspace/upstream/kafo_parsers/lib/kafo_parsers/puppet_strings_module_parser.rb:43:in `initialize'
	 4: from /home/ehelms/workspace/upstream/kafo_parsers/lib/kafo_parsers/puppet_strings_module_parser.rb:138:in `run_puppet'
	 3: from /usr/share/ruby/open3.rb:258:in `capture3'
	 2: from /usr/share/ruby/open3.rb:95:in `popen3'
	 1: from /usr/share/ruby/open3.rb:199:in `popen_run'
/usr/share/ruby/open3.rb:199:in `spawn': wrong first argument (ArgumentError)
```